src/status_im2/common/bottom_sheet/events.cljs

```diff
@@ -0,0 +1,30 @@
+(ns status-im2.common.bottom-sheet.events
+  (:require [utils.re-frame :as rf]))
+
+(rf/defn show-bottom-sheet
+  [{:keys [db]} {:keys [view options]}]
+  {:dispatch                  [:dismiss-keyboard]
```

Comment:
mhm im not sure, what if we'll have a case when we don't want to dismiss a keyboard? but i think in such cases we could open it again in the new sheet, ok